src/components/backButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {TouchableOpacity, StyleSheet} from 'react-native';
+import CloseBlack from '@assets/Cross.svg';
+import BackArrow from '@assets/Back_Arrow.svg';
+
+interface IBackButtonProps {
+  onClick: () => void;
+  isCloseButton?: boolean; // true for cross button else normal back button
```

Comment:
Let make cross a different component not merge it with back button.

Reply:
okay.